.github/workflows/tools.yml

```
@@ -0,0 +1,33 @@
+name: Tools
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always # Force Cargo to use colors
+  TERM: xterm-256color
+  WORKING_DIR: tools/didc-js
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ env.WORKING_DIR }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Setup PNPM"
+        uses: dfinity/ci-tools/actions/setup-pnpm@f241586372f71ae0875545bf5ce2aade82ebc0b5
+        with:
+          working-directory: ${{ env.WORKING_DIR }}
```

Is the built-in `working-directory` not working here?
```suggestion
        working-directory: ${{ env.WORKING_DIR }}
```

---

Apparently, the working-directory does not work for `uses` steps